pkg/checks/external/nodeCheck/main.go

```diff
@@ -66,29 +66,34 @@ func WaitForKubeProxy(client *kubernetes.Clientset, namespace string, ctx contex
 	return nil
 }
 
-// WaitForNodeAge checks the node's age to see if its less than the minimum node age. If so, sleeps for a given sleep duration.
-func WaitForNodeAge(client *kubernetes.Clientset, namespace string, minNodeAge time.Duration, sleepDuration time.Duration, ctx context.Context) error {
+// WaitForNodeAge checks the node's age to see if its less than the minimum node age. If so, sleeps until the node
+// reaches the minimum node age.
+func WaitForNodeAge(client *kubernetes.Clientset, namespace string, minNodeAge time.Duration, ctx context.Context) error {
```

Comment:
when supplying a context, it should always come first in the function signature
```suggestion
func WaitForNodeAge(ctx context.Context, client *kubernetes.Clientset, namespace string, minNodeAge time.Duration) error {
```